### PR TITLE
US151276: adding deduction percentage

### PR DESCRIPTION
**Pull request by emsandrews**
[US151276](https://rally1.rallydev.com/#/?detail=/userstory/697596605061&fdp=true): Hypermedia APIs

adding deduction percentage api for negative grading feature

### CODE REVIEW

File: src/activities/quizzes/QuizEntity.js [hunk 1]
```diff
@@ -182,6 +182,26 @@ export class QuizEntity extends Entity {
 		return entity && entity.hasClass(Classes.quizzes.checked);
 	}
 
+	/**
+	 * @returns {bool} Whether or not the user can edit the Deduction Percentage property
+	 */
+	canEditDeductionPercentage() {
```
Comment (alexli01) [inline src/activities/quizzes/QuizEntity.js:188]: Where is this function used? I only see it used in a test.

File: src/activities/quizzes/QuizEntity.js [hunk 2]
```diff
@@ -602,6 +622,7 @@ export class QuizEntity extends Entity {
 		const updatePasswordAction = this.canEditPassword() ? this._formatUpdatePasswordAction(quiz) : null;
 		const updateNotificationEmail = this.canEditNotificationEmail() ? this._formatNotificationEmailAction(quiz) : null;
 		const updatePreventMovingBackwards = this.canEditPreventMovingBackwards() ? this._formatUpdatePreventMovingBackwards(quiz) : null;
+		const updateDeductionPercentageAction = this.deductionPercentage() ? this._formatUpdateDeductionPercentage(quiz) : null;
```
Comment (alexli01) [inline src/activities/quizzes/QuizEntity.js:625]: Should it be canEditDeductionPercentage? Like how the others were done?

File: src/activities/quizzes/QuizEntity.js (hunk 1 above)
Comment (emsandrews) [reply src/activities/quizzes/QuizEntity.js:188]: updated: method now used on line 625

**Review by semibran — APPROVED** [review]
LGTM

**Review by alexli01 — APPROVED** [review]

**Pull request merged**